source/winAPI/sessionTracking.py

```diff
@@ -84,6 +84,29 @@
 """
 
 
+class _IgnoreWindowsLockState:
+	"""
+	Used to bypass security checks by forcing isWindowsLocked to return False.
+	Should only be used where:
+	- security checks are not possible, such as during NVDA initialization/termination.
+	- and code is safe to execute, there are no exploits even if the lock state is ignored.
+
+	Usage
+	```
+	with _IgnoreWindowsLockState():
+		doCodeWhichIsSafe
+	"""
+	shouldIgnoreLockState = False

```

Comment:
is this design going to be risky if multiple threads are reading from here?
e.g. 1 thread is using `_IgnoreWindowsLockState`, and other thread is not. Both act as if windows is unlocked.